### PR TITLE
Refs Task#405741 Don't show bot key when create/update bot

### PR DESCRIPTION
**Pull request by tamtt-1031**
## Related Tickets

- [#405741](https://dev.sun-asterisk.com/issues/405741)

## What's this PR do ?

- [x] Don't show bot key when create/update bot

## Library
*(List package, library third party add new include version)*

- Library: google-analysis.js

## Performance

- [ ] Resolved n + 1 query
- [ ] Run explain query already
- [ ] Time run rake task : 1000 ms

## Checklist

- [x] It was tested in local success?
- [x] Fill link PR into ticket and the opposite
- [x] Note purpose/reason, solution, scope of influence into ticket

## ENV note

```
```

## Notes
*(Other notes)*
Screenshot from 2019-12-24 11-23-17
Screenshot from 2019-12-24 11-23-42

### CODE REVIEW

**Comment by tamtt-1031** [comment]
ready

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>failure</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr></table>

File: app/Http/Requests/BotUpdateRequest.php [hunk 1]
```diff
@@ -25,22 +25,26 @@ public function authorize()
      */
     public function rules()
     {
-        return [
+        $rules = [
             'name' => [
```
Comment (framgiaciv3) [inline app/Http/Requests/BotUpdateRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/BotUpdateRequest.php</br>**Line**: 29</br>Each line in an array declaration must end in a comma

**Review by nguyenthinhhn — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>phpunit</td><td>failure</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr></table>

**Review by ghost — APPROVED** [review]

**Comment by framgiaciv3** [comment]
No data coverage!

File: app/Http/Requests/BotUpdateRequest.php [hunk 2]
```diff
@@ -25,22 +25,27 @@ public function authorize()
      */
     public function rules()
     {
-        return [
+        $rules =
+        [
             'name' => [
```
Comment (framgiaciv3) [inline app/Http/Requests/BotUpdateRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/BotUpdateRequest.php</br>**Line**: 30</br>Each line in an array declaration must end in a comma

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>failure</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>failure</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr></table>

File: app/Http/Requests/BotUpdateRequest.php (hunk 2 above)
Comment (framgiaciv3) [inline app/Http/Requests/BotUpdateRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Requests/BotUpdateRequest.php</br>**Line**: 30</br>Each line in an array declaration must end in a comma

**Comment by framgiaciv3** [comment]
coverage</br> **1. Result build** </br><table><tr><th>Style</th><th>Status</th><th>Ignored</th></tr><tr><td>cp__env_example__env._start_run_.3</td><td>success</td><td></td></tr><tr><td>php_artisan_key_gene._start_run_.13</td><td>success</td><td></td></tr><tr><td>composer_dump-autolo._start_run_.17</td><td>success</td><td></td></tr><tr><td>phpunit</td><td>success</td><td></td></tr><tr><td>php_artisan_config_c._start_run_.11</td><td>success</td><td></td></tr><tr><td>composer_install._start_run_.5</td><td>success</td><td></td></tr><tr><td>phpcs</td><td>success</td><td></td></tr><tr><td>framgia-ci_test-conn._start_run_.15</td><td>success</td><td></td></tr><tr><td>php_artisan_migrate_._start_run_.19</td><td>success</td><td></td></tr><tr><td>yarn_install._start_run_.7</td><td>success</td><td></td></tr><tr><td>yarn_dev._start_run_.9</td><td>success</td><td></td></tr></table>

**Review by quanghung97 — APPROVED** [review]
OK

**Comment by phucqd-1305** [comment]
OK

**Pull request merged**